assets/migrations/mysql/007_add_collate_index.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+CREATE INDEX idx_collate ON tuple (
+    store,
+    _user,
+    relation,
+    object_type,
+    (object_id COLLATE utf8mb4_bin)
+);
+
+DROP INDEX idx_reverse_lookup_user ON tuple;
+
+-- +goose Down
+DROP INDEX idx_collate on tuple;
+
+CREATE INDEX idx_reverse_lookup_user ON tuple (store, object_type, relation, _user);
```

Comment:
```suggestion
CREATE INDEX idx_reverse_lookup_user ON tuple (store, object_type, relation, _user);

```

nit: please add a trailing newline